01_fib/baijie/fib.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+// fib function to generate fibonacci sequence
+func fib(i int) {
+	// initial values
+	m, n := 0, 1
+	// positive case
+	if i >= 0 {
+		for j := 0; j < i; j++ {
+			m += n
+			m, n = n, m
+			fmt.Fprintln(out, m)
+		}
+		// negative case
```

Comment:
Another comment that I don't think adds much value. If you must, I suggest aligning it with the upper comment for legibility. Personally, I would leave both out.

Reply:
Done